internal/pkg/manifest/manifest_test.go

```diff
@@ -377,7 +377,7 @@ func fullJson() string {
       "relations": [
         {
           "componentId": "keboola.wr-db-mysql",
-          "id": "12",
+          "configId": "12",
```

Comment:
Drobnost: premenoval som `id` -> `configId` v relation, v `manifest.json` ... aby bolo jasne, ze je to ID cieloveho configu, a nie nejake ID relacie.